src/main/java/friday/Task.java

```diff
@@ -0,0 +1,51 @@
+package friday;
+
+/**
+ * Represents a task in the Friday application.
+ */
+public class Task {
+    protected String name;
+    protected boolean isDone;
+
```

Comment:
Should the constructor have its own Javadoc?

```suggestion
/**
 * Constructs a new instance of {@code Task}.
 *
 * @param name The name of the task
 */
```